fonts.css

```diff
@@ -1,18 +1,21 @@
 @font-face {
   font-family: 'Museo';
-  src: url('../fonts/MuseoSansCyrl-500.woff')format('woff'), url('../fonts/MuseoSansCyrl-500.woff2')format('woff2');
+  src: url('fonts/MuseoSansCyrl-500.woff')format('woff'), 
+       url('fonts/MuseoSansCyrl-500.woff2')format('woff2');
   font-weight: 500;
   font-style: normal;
 }
 @font-face {
   font-family: 'PTSerif';
-  src: url('../fonts/PTSerif-Bold.woff')format('woff'),url('../fonts/PTSerif-Bold.woff2')format('woff2');
+  src: url('fonts/PTSerif-Bold.woff')format('woff'),
+       url('fonts/PTSerif-Bold.woff2')format('woff2');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)